spec/tapioca/cli_spec.rb

```diff
@@ -583,19 +583,13 @@ class Foo::Secret
       output = execute("sync")
 
       refute_includes(output, "-- Removing:")
-      refute_includes(output, "++ Adding:")
```

Comment:
I don't think these should change at all.

Reply:
I will revert all the changes back :)